lib/qiita/markdown/filters/greenmat.rb

```diff
@@ -16,13 +20,18 @@ def greenmat
             Qiita::Markdown::Greenmat::HTMLRenderer.new(hard_wrap: true, with_toc_data: true),
             autolink: true,
             fenced_code_blocks: true,
-            footnotes: true,
+            footnotes: options[:footnotes],
             no_intra_emphasis: true,
             no_mention_emphasis: true,
             strikethrough: true,
             tables: true,
           )
         end
+
+
+        def options
+          @options ||= DEFAULT_OPTIONS.merge(context[:markdown] || {})
+        end
```

Comment:
I agree, it would be simpler, but in `filters/truncate.rb`, this is configurable in both `Processor` and `Filters::Truncate`. 
For consistency, in [this line](https://github.com/increments/qiita-markdown/blob/master/lib/qiita/markdown/filters/truncate.rb#L75-L77) of `truncate.rb` should be removed i think. (but i dont think the change should be in this pull request)